src/internet_identity/src/storage.rs

```diff
@@ -106,16 +113,137 @@ const STABLE_MEMORY_RESERVE: u64 = 8 * GB / 10;
 
 const PERSISTENT_STATE_MAGIC: [u8; 4] = *b"IIPS"; // II Persistent State
 
+/// MemoryManager parameters.
+const ANCHOR_MEMORY_ID: MemoryId = MemoryId::new(0);
+const BUCKET_SIZE_IN_PAGES: u16 = 1024;
```

Comment:
Why did you chose this bucket size? Maybe add a comment here to explain.

Also, 1024 is quite large, the (new) default is 128. We might end up wasting quite some space, if we allocate `VirtualMemory`s for smallish things (e.g. asset storage, fixed persistent state, etc.).

Reply:
Yep, this is up for discussion -- let's chat and I'll change it afterwards.

Reply:
As discussed, I changed the bucket size to 128 (and added a comment).